yarn.lock

```diff
@@ -3676,13 +3676,13 @@ eslint-scope@^4.0.0:
     estraverse "^4.1.1"
 
 eslint-utils@^1.3.1:
-  version "1.4.0"
-  resolved "https://nexus.global.picnicinternational.com/repository/npm-public/eslint-utils/-/eslint-utils-1.4.0.tgz#e2c3c8dba768425f897cf0f9e51fe2e241485d4c"
-  integrity sha512-7ehnzPaP5IIEh1r1tkjuIrxqhNkzUJa9z3R92tLJdZIVdWaczEhr3EbhGtsMrVxi1KeR8qA7Off6SWc5WNQqyQ==
+  version "1.4.3"
+  resolved "https://registry.yarnpkg.com/eslint-utils/-/eslint-utils-1.4.3.tgz#74fec7c54d0776b6f67e0251040b5806564e981f"
+  integrity sha512-fbBN5W2xdY45KulGXmLHZ3c3FHfVYmKg0IrAKGOkT/464PQsx2UeIzfz1RmEci+KLm1bBaAzZAh8+/E+XAeZ8Q==
```

Comment:
@nov1n since this repository is public, perhaps we should replace all references to the internal Nexus deployment here.

Reply:
@larslockefeer Do you have any thoughts on this? I'm not very familiar with yarn.

Reply:
The problem with changing the references to the internal Nexus deployment seems to be that the project depends on `picnic-react-mise-en-place`. I am unsure if this dependency is/can be hosted elsewhere?

Reply:
Good point. CC @larslockefeer.